packages/ethereum/contracts/HoprChannels.sol

```diff
@@ -482,13 +388,13 @@ contract HoprChannels is IERC777Recipient, ERC1820Implementer {
         require(counterparty != address(0), "counterparty must not be empty");
 
         (,,, Channel storage channel) = _getChannel(opener, counterparty);
-        require(channel.deposit > 0, "channel must be funded");
+        require(channel.partyABalance + channel.partyBBalance > 0, "channel must be funded");
```

Comment:
`channel.partyABalance.add(channel.partyBBalance)` to avoid overflow

Reply:
thanks